config/config_test.go

```diff
@@ -3190,64 +3190,66 @@ func TestTCF2FeatureOneVendorException(t *testing.T) {
 }
 
 func TestIsAccountEventEnabled(t *testing.T) {
+	boolFalse := false
+	boolTrue := true
```

Comment:
Nitpick: We don't need these two if we simply use the `ToPtr()` function found in `util/ptrutil/ptrutil.go`